database/src/main/java/info/metadude/android/eventfahrplan/database/extensions/SQLiteDatabaseExtensions.kt

```diff
@@ -29,6 +29,22 @@ fun SQLiteDatabase.read(
         limit
 )
 
+fun SQLiteDatabase.updateRow(
+        tableName: String,
+        contentValues: ContentValues,
+        columnName: String? = null,
+        columnValue: String? = null
```

Comment:
These are never `null` when this method is called. If feels premature to add support for this if the functionality is not used. 
You might introduce a bug in on of the unused branches. You won't notice because there are no tests covering this (and the code is not used in production). It won't be a problem until you add a feature actually using this code. When things then don't work as expected this will be the last place you'll be looking for a bug. You'll suspect the newly added code, not the code that has been there for a long time not causing any problems (because it wasn't used; but you won't remember that).

Reply:
A totally agree. :heavy_check_mark: Changed in 6bb13849d4f9c15c2c7611fa79685fc221de412e.